### PR TITLE
Fix to read the correct log level settings

### PR DESCRIPTION
**Pull request by WilliamBZA**

### CODE REVIEW

**Comment by danielmarbach** [comment]
How about this? https://github.com/Particular/ServiceControl/blob/master/src/ServiceControl/Infrastructure/Settings/LoggingSettings.cs#L28-L32

File: src/ServiceControl.Audit/Infrastructure/Settings/LoggingSettings.cs [hunk 1]
```diff
@@ -25,8 +25,8 @@ public LoggingSettings(string serviceName, LogLevel defaultLevel = null, LogLeve
 
         LogLevel InitializeLevel(string key, LogLevel defaultLevel)
         {
-            var levelText = SettingsReader<string>.Read("ServiceControl.Audit", key);
```
Comment (danielmarbach) [inline src/ServiceControl.Audit/Infrastructure/Settings/LoggingSettings.cs:28]: Ah so the `key` was used as a default value and `ServiceContro.Audit` as the name 🥇

**Review by danielmarbach — APPROVED** [review]

File: src/ServiceControl.Audit/Infrastructure/Settings/LoggingSettings.cs (hunk 1 above)
Comment (WilliamBZA) [reply src/ServiceControl.Audit/Infrastructure/Settings/LoggingSettings.cs:28]: Yes, this was a real champion refactoring I did that introduced this bug. In all 3 SC instances too.

**Review by mikeminutillo — APPROVED** [review]

**Comment by WilliamBZA** [comment]
Closing and re-creating to try get TC to report build status

**Pull request closed without merging**